tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0007_event_sink.py

```diff
@@ -79,6 +79,14 @@ def downgrade():
         "DROP TABLE IF EXISTS {{ ASPECTS_EVENT_SINK_DATABASE }}.course_blocks"
         f"{on_cluster}"
     )
+    op.execute(
```

Comment:
I think this needs to be in a new migration otherwise existing installs won't get it since they've already run this migration.